tests/strings/test_levenshtein_distance.py

```diff
@@ -0,0 +1,17 @@
+from strings.levenshtein_distance import levenshtein_distance
+
+
+def test_levenshtein_distance_between_test_and_string():
```

Comment:
To have a consistent naming:
```suggestion
def test_levenshtein_distance_between_test_and_data():
```